workspaces/adventure-pack/goodies/python3/src/is_palindrome/__init__.py

```diff
@@ -0,0 +1,9 @@
+def is_palindrome(arr: list | str) -> bool:
```

Comment:
One disadvantage of a name like `arr` is that it suggests a list variable. However, as we are supporting string as well (thank you!) consider a name like `sequence` that is more generalized.

Reply:
Yeah, I just named it arr cause couldn't figure out what to name it, and strings are kinda like arrays. Ill update it now.

Reply:
Should pass the tests now as well. I ended up making a lot of commits, but not sure if that's a problem or how to merge them.

Reply:
It's not a problem, they will be squashed into a single commit when the PR is merged 🙂